app/views/alchemy/pages/show.rb

```diff
@@ -131,6 +132,22 @@ def google_analytics_code
         ENV.fetch('GOOGLE_ANALYTICS_CODE', 'UA-123456-1')
       end
 
+      def as_json
+        {
+          url: page_object.url,
```

Comment:
Some of these calls to methods on `page_object` are followed by `&.` others just `.` -- keep it consistent.